### DISCUSSION TITLE
补帧时播放杜比视界

### DISCUSSION THREAD

**Discussion by bulesky89857** [hooke007/mpv_PlayKit#156] [mpv 通用]
播放杜比视界视频偏色，请问如何处理，网上有其它mpv diy版可以正常播放杜比视频，但是没有补帧滤镜。所以想是不是可以融合一下。

**Comment by hooke007** [comment]
https://github.com/hooke007/MPV_lazy/wiki/0_FAQ#%E6%9D%9C%E6%AF%94%E8%A7%86%E7%95%8C%E5%81%8F%E8%89%B2

**Reply by bulesky89857** [reply]
谢谢，确实不偏色了，但是和补帧滤镜冲突，打开补帧偏色，请问有办法吗？

**Reply by hooke007** [reply]
暂无，dv的处理也是滤镜，传递给vs后应该是丢数据了
https://github.com/mpv-player/mpv/issues/10438

**Reply by bulesky89857** [reply]
好的，明白了，谢谢。

**Reply by hooke007** [reply]
等待上游合并 https://github.com/mpv-player/mpv/pull/12593
